### PR TITLE
Added new functions for Keithley2400 

### PR DESCRIPTION
**Pull request by paulgoulain**
Added two missing functions for the Keithley2400 driver : 
- Filters
- Output state

### CODE REVIEW

File: pymeasure/instruments/keithley/keithley2400.py [hunk 1]
```diff
@@ -303,6 +303,83 @@ class Keithley2400(Instrument, KeithleyBuffer):
...
+
+    #####################
+    # Output subsystem #
+    #####################
+
+    current_output_off_state = Instrument.control(
+        ":OUTP:CURR:SMOD?", ":OUTP:CURR:SMOD %s",
+        """ Select the output-off state of the SourceMeter.
+        HIMP : output relay is open, disconnects external circuitry.
+        NORM : V-Source is selected and set to 0V, Compliance is set to 0.5%
```
Comment (bilderbuchi) [inline pymeasure/instruments/keithley/keithley2400.py:359]: Is the meaning of the various parameters really identical between current_ and voltage_ versions?

**Review by bilderbuchi — CHANGES_REQUESTED** [review]
Thanks for this, I had a quick look.

File: pymeasure/instruments/keithley/keithley2400.py (hunk 1 above)
Comment (paulgoulain) [reply pymeasure/instruments/keithley/keithley2400.py:359]: The documentation is quite shallow on this function. I added it after someone requested it (see issue #319). From what I understand, depending on which output is currently selected, the Keithley will switch to right kind of off state mode. I don't know how to test it though to confirm the behavior.

**Review by bilderbuchi — APPROVED** [review]

File: pymeasure/instruments/keithley/keithley2400.py (hunk 1 above)
Comment (bilderbuchi) [reply pymeasure/instruments/keithley/keithley2400.py:359]: OK then.

**Pull request merged**